dockers/docker-database-global/global_db.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+import swsssdk
+import json
+globaldb = swsssdk.SonicV2Connector(host='redis_global.server', port='6385')
```

Comment:
how is this 6385 decided?

Reply:
it is a IANA unassigned port, should be good for internal use. Also I made the global db port number configurable through globaldb.conf